raiden/tests/test_service.py

```diff
@@ -29,6 +29,32 @@ def test_ping(raiden_network):
     assert decoded.echo == sha3(ping.encode() + app1.raiden.address)
 
 
+@pytest.mark.timeout(3)
```

Comment:
I see. I understand now. Checked out the PR locally and ran the test both with your fix and without. Thanks.